### DISCUSSION TITLE
Can't get it to work anymore

### DISCUSSION THREAD

**Discussion by BerjanSchroer** [fabianishere/udm-iptv#314] [Q&A]
Hi everyone.

This worked like a charm the past few years. but now i have a problem after an update of the UDM. I have re-installed this, but it is not working ;(

ISP -- KPN 

Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br50
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
=== Service Logs ===
Mar 16 16:45:19 KKH-UDM-01 udm-iptvd[830605]: DEBUG[init_interface@208]: if_mtu = 1500
Mar 16 16:45:19 KKH-UDM-01 udm-iptvd[830605]: DEBUG[init_interface@214]: if_falgs = 4163
Mar 16 16:45:19 KKH-UDM-01 udm-iptvd[830605]: DEBUG[init_interface@178]: iptv's ipv4 address is 10.88.101.67
Mar 16 16:45:19 KKH-UDM-01 udm-iptvd[830605]: DEBUG[init_interface@195]: if_index = 224072
Mar 16 16:45:19 KKH-UDM-01 udm-iptvd[830605]: DEBUG[init_interface@208]: if_mtu = 1500
Mar 16 16:45:19 KKH-UDM-01 udm-iptvd[830605]: DEBUG[init_interface@214]: if_falgs = 4163
Mar 16 16:45:19 KKH-UDM-01 udm-iptvd[830605]: ERROR[main@619]: exiting.........
Mar 16 16:45:19 KKH-UDM-01 IMPROXY[830605]: ERROR[main@619]: exiting.........
Mar 16 16:45:19 KKH-UDM-01 systemd[1]: udm-iptv.service: Main process exited, code=exited, status=1/FAILURE
Mar 16 16:45:19 KKH-UDM-01 systemd[1]: udm-iptv.service: Failed with result 'exit-code'.

What to do?

Kind regards, 
Berjan

**Comment by anagorn** [comment]
I experience the same problem; running on a separat VLAN.
Reconfiguring doesn't seem to help

**Comment by horolux** [comment]
Hi, 

I experience the same problem after an update of my UDM-SE as well as the Network application on it. Here are both versions:

UDM-SE                        : UniFi OS 3.2.12
Network application    : 8.1.113

**Comment by horolux** [comment]
@fabianishere :  is there a solution for this problem?

**Comment by BerjanSchroer** [comment]
Hi, Anyone still having trouble? 

Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br55
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
=== Service Logs ===
May 28 09:58:21 KKH-UDM-01 udm-iptvd[1857151]: DEBUG[init_interface@208]: if_mtu = 1500
May 28 09:58:21 KKH-UDM-01 udm-iptvd[1857151]: DEBUG[init_interface@214]: if_falgs = 4163
May 28 09:58:21 KKH-UDM-01 udm-iptvd[1857151]: DEBUG[init_interface@178]: iptv's ipv4 address is 10.88.101.197
May 28 09:58:21 KKH-UDM-01 udm-iptvd[1857151]: DEBUG[init_interface@195]: if_index = 668
May 28 09:58:21 KKH-UDM-01 udm-iptvd[1857151]: DEBUG[init_interface@208]: if_mtu = 1500
May 28 09:58:21 KKH-UDM-01 udm-iptvd[1857151]: DEBUG[init_interface@214]: if_falgs = 4163
May 28 09:58:21 KKH-UDM-01 udm-iptvd[1857151]: ERROR[main@619]: exiting.........
May 28 09:58:21 KKH-UDM-01 IMPROXY[1857151]: ERROR[main@619]: exiting.........
May 28 09:58:21 KKH-UDM-01 systemd[1]: udm-iptv.service: Main process exited, code=exited, status=1/FAILURE
May 28 09:58:21 KKH-UDM-01 systemd[1]: udm-iptv.service: Failed with result 'exit-code'.

**Comment by jdjongh** [comment]
It there an update on this topic?
